krummstab/__init__.py

```diff
@@ -365,6 +367,197 @@ def unsupported_marking_mode_error(_the_config: config.Config) -> None:
     logging.critical(f"Unsupported marking mode {_the_config.marking_mode}!")
 
 
+# ============================== Summarize Sub-Command =========================
+
+
+def summarize_points_per_exercise(_the_config: config.Config) -> None:
```

Comment:
This is a very long function. You can extract some functionality like loading the score files into smaller ones.